src/server.ts

```diff
@@ -5,19 +5,40 @@ import sequelize from "./config/mySQLConf";
 import initializeDatabase from "./initializeDatabase";
 
 const PORT = process.env.PORT || 3000;
-const DB_USER = process.env.DB_USER || "root";
-const DB_PASSWORD = process.env.DB_PASSWORD;
-const DB_HOST = process.env.DB_HOST || "localhost";
 export const JWT_SECRET = process.env.JWT_SECRET!;
+
```

Comment:
Just a suggestion you might do it in another PR, instead of having commented code, maybe use the env.NODE_ENV and switch between codes accordingly.